Assets/Global/Scripts/Enemies/RangedEnemy/RangedEnemy.cs

```diff
@@ -0,0 +1,24 @@
+
+using UnityEngine;
+namespace EnemiesNS
+{
+    public class RangedEnemy : EnemyBase
+    {
+        public Transform bulletSpawnPoint; 
```

Comment:
can these vars get [Tooltip("explanation about the var")] to keep them in line with the rest of the enemy vars, they should all have tooltips and atleast kevin seemed to like them

Reply:
would be nice